README.md

```diff
@@ -29,7 +29,7 @@ first time.
 System Requirements
 -------------------
 
-This project depends on Python 2.7 and the Mercurial >= 4.6
+This project depends on Python 2.7 or 3.5+, and the Mercurial >= 4.6
```

Comment:
Perhaps it should say: "... and the Mercurial package >= 4.6 (>= 5/2, if Python 3.5+)" to avoid future support issues.